asm/__osGetFpcCsr.s

```diff
@@ -9,7 +10,8 @@
 
 .balign 16
 
-glabel __osGetFpcCsr
-/* 008680 80007A80 4442F800 */  cfc1  $v0, $31
-/* 008684 80007A84 03E00008 */  jr    $ra
-/* 008688 80007A88 00000000 */   nop   
+BEGIN __osGetFpcCsr
+    cfc1    $v0, FpCsr
```

Comment:
Is it `FpcCsr` as in `__osGetFpcCsr`, or `FpCsr`?

Fpc may stand for Floating Point Coprocessor
and Csr for Control/Status Register?

Reply:
`FpcCsr`, will change

Reply:
I noticed in the fault.c PR, `Fault_PrintFPCSR` calls the register `FPCSR`
also a bunch of defines use `FPCSR`

are you changing all of this to fpc csr then?

Reply:
I ended up not doing this since I've now switched to using `C1_FPCSR` from `r4300.h`